tests/pipelines/test_pipelines_text_to_audio.py

```diff
@@ -173,6 +174,60 @@ def test_vits_model_pt(self):
         outputs = speech_generator(["This is a test", "This is a second test"], batch_size=2)
         self.assertEqual(ANY(np.ndarray), outputs[0]["audio"])
 
+    @slow
+    @require_torch
+    def test_forward_model_kwargs(self):
+        # use vits - a forward model
+        speech_generator = pipeline(task="text-to-audio", model="kakao-enterprise/vits-vctk", framework="pt")
+
+        # for reproducibility
+        set_seed(555)
+        outputs = speech_generator("This is a test", forward_params={"speaker_id": 5})
+        audio = outputs["audio"]
+
+        with self.assertRaises(TypeError):
+            # assert error if generate parameter
+            outputs = speech_generator("This is a test", forward_params={"speaker_id": 5, "do_sample": True})
```

Comment:
is there any better example of forward params? 
Speaker id seems to me that it could be part of a generation config (but also is not a tensor and the pr seems to say that we could have tensors here).

Reply:
unfortunately I can only think of Vits and `speaker_id` is basically the only optional parameter of the forward pass. BTW,  tensor params are tested by `test_conversion_additional_tensor`